spec/javascripts/analytics/ecommerce.spec.js

```diff
@@ -298,6 +298,89 @@ describe('Ecommerce reporter for results pages', function() {
     })
   });
 
+  it('tracks clicks with different event labels', function() {
+    element = $('\
+      <div> \
+        <div data-analytics-ecommerce data-list-title="First list" data-ecommerce-start-index="1" data-search-query="search query">\
+          <div \
+            data-ecommerce-row=1\
+            data-ecommerce-path="/path/to/page"\
+            data-ecommerce-content-id="AAAA-1111">\
+          </div>\
+        </div>\
+        <div data-analytics-ecommerce data-list-title="Second list" data-track-click-label="Custom click label" data-ecommerce-start-index="1" data-search-query="blah">\
+          <div \
+            data-ecommerce-row=1\
+            data-ecommerce-path="/path/to/blah"\
+            data-ecommerce-content-id="AAAA-2222">\
+          </div>\
+        </div>\
+      </div>\
+    ');
+
+    GOVUK.Ecommerce.start(element.find('[data-analytics-ecommerce]'));
+    element.find('[data-ecommerce-row]').click();
+
+    expect(ga).toHaveBeenCalledWith('ec:addProduct', {
+      id: 'AAAA-1111',
+      position: 1,
+      dimension71: 'search query'
+    });
+    expect(ga).toHaveBeenCalledWith('ec:addProduct', {
+      id: 'AAAA-2222',
+      position: 1,
+      dimension71: 'blah'
+    });
+    expect(ga).toHaveBeenCalledWith('ec:setAction', 'click', {list: 'First list'})
+    expect(ga).toHaveBeenCalledWith('ec:setAction', 'click', {list: 'Second list'})
+    expect(ga).toHaveBeenCalledWith('send', {
```

Comment:
Does the test need to `expect` all of the GA calls? It might be shorter and clearer if it only expected the ones that are being explicitly tested here.

Reply:
let me just put one and see what happens :)

Reply:
I'm not 100% this morning but it looks like you've removed the relevant ones and left the non-relevant ones? Where's the tests for 'first list' and 'second list'?